2-mandatory/2-water-bottle.js

```diff
@@ -15,22 +15,26 @@ You have to implement the missing features according to the specification.
 */
 
 // Here is your starting point:
-let bottle = {
+const UNITS = 10;
+const bottle = {
   volume: 0,
-  fillUp: function () {
-    // calling this function should completely fill your bottle (volume = 100);
+  capacity: 100,
+  fillUp() {
+    this.volume = 100;
   },
-  pour: function () {
-    // calling this function should increase your bottle volume by 10 units;
+  pour() {
+    if (this.isFull()) return;
+    this.volume = this.volume + UNITS > this.capacity ? this.capacity : this.volume + UNITS;
   },
-  drink: function () {
-    // calling this function should decrease your bottle volume by 10 units;
+  drink() {
+    if (this.isEmpty()) return;
+    this.volume = this.volume + UNITS < 0 ? 0 : this.volume - UNITS;
```

Comment:
You probably meant the following?
```suggestion
    this.volume = this.volume - UNITS < 0 ? 0 : this.volume - UNITS;
```